qemu/tests/cfg/snp_attestation.cfg

```diff
@@ -0,0 +1,43 @@
+- snp_attestation:
+    type = snp_basic_config
+    only Linux
+    kill_vm = yes
+    login_timeout = 240
+    start_vm = no
+    image_snapshot = yes
+    mem = 8192
+    smp = 8
+    required_qemu = [9.1.0, )
+    vm_secure_guest_type = snp
+    vm_sev_reduced_phys_bits = 1
+    vm_sev_cbitpos = 51
+    virtio_dev_disable_legacy = on
+    vm_mem_backend = memory-backend-memfd
+    bios_path =  /usr/share/edk2/ovmf/OVMF.amdsev.fd
+    snp_module_path = "/sys/module/kvm_amd/parameters/sev_snp"
+    module_status = Y y 1
+    snp_guest_check = "journalctl|grep -i -w snp"
+    guest_tool_install = "dnf install -y snpguest"
+    snpguest_sourcebuild = 1
+    attestation_script = regular_attestation_workflow.sh
+    snpguest_install_script = snpguest_install.sh
+    guest_dir = /home
+    guest_cmd = ${guest_dir}/${attestation_script}
+    host_script = sev-snp/${attestation_script}
+    snpguest_buildcmd = "${guest_dir}/${snpguest_install_script} --repo https://github.com/virtee/snpguest.git --tag v0.9.1"
+    snpguest_build_location = sev-snp/${snpguest_install_script}
+    variants:
+        - policy_default:
+            snp_policy = 196608
```

Comment:
Seems we are switch to hex format?

Reply:
Sure, will be taken care in the revised push.